tests/query/snql/test_query.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from snuba import state
+from snuba.datasets.factory import get_dataset
+from snuba.query.expressions import Column, FunctionCall, Literal
+from snuba.query.logical import Query, SelectedExpression
+from snuba.query.snql import parse_snql_query
+
+test_cases = [
+    pytest.param(
+        "COLLECT 4-5, 3*g(c), c",
```

Comment:
Shouldn't the grammar reject a query that does not have a MATCH clause ?
Even if you do not parse the content of the match, the query should still be well formatted.